azuredeploy.json

```
@@ -288,6 +294,18 @@
         "scope": "[concat(subscription().id, '/resourceGroups/',  resourceGroup().name, '/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('droneSchedulerIdName'))]"
       }
     },
+    {
+      "type": "Microsoft.ManagedIdentity/userAssignedIdentities/providers/roleAssignments",
+      "name": "[concat(parameters('ingestionIdName'), '/Microsoft.Authorization/', guid('msi-ingestion', resourceGroup().id))]",
+      "apiVersion": "2021-04-01-preview",
+      "comments": "Grant the AKS cluster Managed Identity access to the ingestion managed id",
+      "properties": {
+        "roleDefinitionId": "[variables('managedIdentityOperatorRoleId')]",
+        "principalId": "[reference(resourceId('Microsoft.ContainerService/managedClusters', variables('aksClusterName')), '2020-03-01').identityProfile.kubeletidentity.objectId]",
+        "principalType": "ServicePrincipal",
+        "scope": "[concat(subscription().id, '/resourceGroups/',  resourceGroup().name, '/providers/Microsoft.ManagedIdentity/userAssignedIdentities/',parameters('ingestionIdName'))]"
```

```suggestion
        "scope": "[resourceId('Microsoft.ManagedIdentity/userAssignedIdentities', parameters('ingestionIdName'))]"
```

I know the rest of them were the way you had it as well, but we should rarely build resource IDs via the `concat()` function, that's a discouraged practice.